python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -108,11 +110,18 @@ def _derive_autoscaling_config_from_ray_cr(ray_cr: Dict[str, Any]) -> Dict[str,
     if IDLE_SECONDS_KEY in autoscaler_options:
         idle_timeout_minutes = autoscaler_options[IDLE_SECONDS_KEY] / 60.0
     else:
-        idle_timeout_minutes = 5.0
-    if autoscaler_options.get(UPSCALING_KEY) == UPSCALING_VALUE_AGGRESSIVE:
-        upscaling_speed = 1000  # i.e. big
+        idle_timeout_minutes = 1.0
```

Comment:
The result from line 111 can end up being 0 due to rounding down.

Reply:
I don't think that's true.
From the CRD definition, the field being divided is an integer. Doing `/` by a float in Python3 does float division.

However, I don't think we currently validate that the integer is positive -- we can do that later in KubeRay.